bundle.js

```diff
@@ -51,17 +51,17 @@
           });
         }
         displayNewsSummary() {
-          const headlines = this.model.getHeadlines();
-          headlines.forEach((headline) => {
-            this.displayHeadline(headline);
+          const newsData = this.model.getNewsInfo();
```

Comment:
Generally speaking, this is very important and well done for doing this. The future you or any other dev looking at the codebase will thank you for being clear.